lib/cli/behaviour/defaults.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const crypto = require('crypto');
+
+module.exports = {
+  dir: process.cwd(),
+  force: false,
+  name: 'default-behaviour',
+  secret: crypto.randomBytes(64).toString('hex')
```

Comment:
What is `secret` for in this context? I suspect a relic of copy-pasting the `init` code.